internal/pkg/manifest/manifest.go

```diff
@@ -381,3 +400,22 @@ func (m *Manifest) sortRecords() {
 		return a.Value().(model.Record).SortKey(m.SortBy) < b.Value().(model.Record).SortKey(m.SortBy)
 	})
 }
+
+func validateVersion(version int) error {
+	if version < 1 || version > 2 {
+		return fmt.Errorf(`unknown version "%d" found in manifest.json`, version)
+	}
+
+	if version == 1 {
+		err := utils.NewMultiError()
+		err.AppendRaw(`Your project needs to be migrated to the new version of the Keboola CLI.`)
```

Comment:
Ak sa pouziva stara verzia, tak vypiseme instrukcie.
Kedze to bola zatial `private beta`, tak som zvolil najrychlejsie rieseni.